bluesky/callbacks/core.py

```diff
@@ -126,8 +127,7 @@ def __init__(self, y, x=None, legend_keys=None, xlim=None, ylim=None,
         else:
             self.x = None
         self.y, *others = _get_obj_fields([y])
-        self.fig = fig
-        self.ax = fig.gca()
+        self.ax = plt.gca()
```

Comment:
I think we should do some `isinstance` checking here to see if it is a figure and raise a warning.  I am 100% sure that this kwarg is being used on the floor.